apps/devhub/tests/test_views.py

```diff
@@ -2483,6 +2483,26 @@ def test_unlisted_addon_pass_validation_with_flag(self, mock_sign_file):
         assert file_.status == amo.STATUS_LITE
         assert mock_sign_file.called
 
+    @mock.patch('devhub.views.sign_file')
+    def test_beta_addon_pass_validation_with_flag(self, mock_sign_file):
+        """Files that pass validation are automatically signed/reviewed if
+        the flag is enabled."""
+        self.create_flag('automatic-validation')
+        version = self.addon.latest_version
+        version.all_files[0].update(status=amo.STATUS_BETA)
```

Comment:
I don't think you need that. I think it's possible to have a beta file for a version that has other non-beta files. Not sure if that makes sense, but in any case I don't think this line is useful.

Also, not setting the current files as beta might fix this test: an addon with no reviewed (full or prelim, for non-beta) files is detected as unreviewed.